Project.toml

```diff
@@ -24,9 +25,8 @@ StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Clarabel = "0.3"
```

Comment:
how do we test QPs if clarabel is removed?

Reply:
I don't think it is necessary in the compat section, it is in the extras section. will see if anything breaks

Reply:
extras section does not list the version, which is the main problem. This doesn't technically break anything but the breakage will appear once clarabel grows.

(Also the package without bounds will get rejected by General. :D )